### ISSUE TITLE
Catalina: "Font download" requests (perhaps Catalina related)

### ISSUE THREAD

**Issue by hrbrmstr** [rstudio/rstudio#4985]
macOS Catalina betas, rstudio dailies, yada yada

I forgot to do this the other day and I'll need to update this again once the next Catalina beta comes out but on a new Catalina beta update, the first time an already installed and used RStudio daily app is opened these appear:

![image](https://user-images.githubusercontent.com/509878/59807710-0f0a6f00-92c7-11e9-9df3-f214c5bafbf6.png)
![image](https://user-images.githubusercontent.com/509878/59807714-1467b980-92c7-11e9-8ba8-7382569aab68.png)

Doing a case insensitive `ripgrep` for `kaiti` in the `rstudio` source tree comes up bupkis (which you may have already figured wld be the case).

BUT, IIRC you're using the new chrome rendering thingy and a google for `chromium kaiti` suggests that might be where this "need" is coming from. Hopefully ^^ helped narrow this down a bit.

**Comment by hrbrmstr** [comment]
possibly related: https://vihung.wordpress.com/2017/05/18/how-to-fix-applications-asking-to-download-fonts-after-macos-sierra-upgrade/

**Comment by ronblum** [comment]
@hrbrmstr Thank you for filing all these what appear to be Catalina issues! I can't reproduce this one with 1.3.264 or 1.2.1551 on Mojave, so it's likely Catalina.

**Comment by ronblum** [comment]
On the first Catalina MacOS 10.15 public beta, I can reproduce this on RStudio Desktop 1.2.1551 and 1.3.278.

**Comment by kevinushey** [comment]
I saw this too.

**Comment by jmcphers** [comment]
I also saw this one, but someone (@ronblum?) has seen other apps also prompt for these fonts.

**Comment by ronblum** [comment]
@jmcphers Yes—I got the font requests when I ran `shinytest::recordTest` in standalone R. I haven't seen it yet in non-R-related applications, but I'll try out some more.

**Comment by jmcphers** [comment]
A hint from the console output:

```
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiSC-Regular"
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiTC-Regular"
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiTC-Regular"
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiSC-Regular"
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiTC-Regular"
QCoreTextFontDatabase: Failed to resolve family name for PostScript name  "STKaitiTC-Regular"
```

Perhaps this happens when we are building our font list at startup?

**Comment by ronblum** [comment]
@hrbrmstr and @jmcphers This problem is resolved with MacOS 10.15 Catalina public beta 3, so I'm closing this. Please update this ticket, though, if you still see the issue.
